dev_guide/authentication.adoc

```diff
@@ -103,3 +114,20 @@ for user input to confirm (`y/N` kind of input) about connecting insecurely.
 
 CLI configuration files allow you to easily
 link:../cli_reference/manage_cli_profiles.html[manage multiple CLI profiles].
+
+// tag::systemadminlogin[]
+
+[NOTE]
+====
+If you have access to system administrator credentials but have logged out of
+the link:../architecture/core_concepts/projects_and_users.html#users[default
+system user] *system:admin*, you can log back in as this user with the following
+command as long as the credentials are still present in your *_~/.kube/config_*
```

Comment:
Was nonspecific about the location of the generated admin credentials for brevity's sake in this note. But maybe that just adds more confusion.

On my OSE 3.0.2 cluster I tried it on, I don't have a $KUBECONFIG set AFAICT, but my ~/.kube/config file seems to have all the system:admin creds, and the above cmd worked fine after "oc logout"ing.